nuget/CodeContracts.MSBuild/build/CodeContracts.MSBuild.props

```diff
@@ -0,0 +1,155 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!--=====================================================================
+        CODE CONTRACT DEFAULTS.
+        All defaults are set as they were by the late Visual Studio extension,
+        except for enabling the static checker by default (otherwise the user
```

Comment:
Some users want runtime checking but not static checking

Reply:
Correct, and they can set the options as they want.